### PR TITLE
Web page.

### PR DESCRIPTION
**Pull request by nati2323**
## First Capstone PR
 - Created a webpage with three different pages.
 
                     - The home page.
                     - The Tickets page.
                     - The about page.
## Links
- The home page is linked in other page.
- The logo in the header links to the home page.

### CODE REVIEW

File: tickets.html [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+      <li><a href="#" class="text-decoration-none color-white">Log out</a></li>
+    </ul>
+
+    <div class="second-nav-container display-flex padding-left-10-percent">
+      <a href="index.html"><img class="logo-image" src="./assets/logo.png" alt="logo image"></a>
+      <ul class="second-nav display-flex justify-content-flex-end list-style-none">
+        <li><a href="index.html" class="text-decoration-none color-black">Home</a></li>
+        <li><a href="tickets.html" class="text-decoration-none color-orange">Tickets</a></li>
+        <li><a href="about.html" class="text-decoration-none color-black">About</a></li>
+      </ul>
```
Comment (CollinsTatang) [inline tickets.html:24]: - [ ] Please add the missing nav tag to your header to match the project design
header

- [ ] Please your hamburger menu should change to a close button when click
### **Your mobile menu when click**
close

File: index.html [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+     <div class="footer-bottom-div display-flex flex-direction-row padding-left-right-10-percent">
+       <div class="footer-image-container">
+         <a href="index.html"><img class="logo-image margin-auto" src="./assets/logo.png" alt="Logo image"></a>
+       </div>
+       <div>
+         <h3>CStadium Show on Saturday, May 28 , 2022 at SoFi Stadium</h3>
+         <p>Please feel free to contact us or any of our dearlships, we will be very happy to hear from you..</p>
+       </div>
+      </div>
+     </footer>
```
Comment (CollinsTatang) [inline index.html:242]: Please try to fix your `Contributors'` section and the `Footers` logo to match the project  [requirement design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
footers

File: about.html [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+        <h1 class="text-align-center color-d3d3d3  underline-red">Contributors</h1>
+
+        <div class="display-flex flex-direction-row width-50-percent footer-top-div justify-content-space-around padding-left-right-10-percent">
+          <i class="fas fa-drum color-d3d3d3"></i>
+          <i class="fab fa-wordpress color-d3d3d3"></i>
+          <i class="fas fa-trademark color-d3d3d3"></i>
+          <i class="fas fa-ad color-d3d3d3"></i>
+          <i class="fas fa-guitar color-d3d3d3"></i>
+        </div>
+      </div>
```
Comment (CollinsTatang) [inline about.html]: - [ ] Kindly remove the contributors from the about page it's not supposed to be there according to the [project design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)

- [ ] Please kindly do the same to the tickets page too

File: index.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
+/* eslint-disable */ 
+function appear() {
+  const readMoreButton = document.getElementById('read-more-button');
+  const additionalSingers = document.querySelectorAll('.additional-singers-article');
+
+  // var arrowForward = document.getElementById('arrowForward');
+  // var arrowBackward = document.getElementById('arrowBackward').innerHTML('22');
```
Comment (CollinsTatang) [inline index.js]: - [ ] Please remove all commented code from your codebase to make it easy to read and maintain

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @nati2323, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Suggested changes
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by bimbolabuari — APPROVED** [review]
Hi @nati2323, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**